### DISCUSSION TITLE
Correlations across age classes

### DISCUSSION THREAD

**Discussion by FisheryScience89** [sdmTMB/sdmTMB#328] [Q&A] [answered]
Hi,

If I'm interested in a spatial-only model with correlated age classes, would it be sensible to use age as the "time" variable? It makes sense in my brain, but maybe I'm being too creative by trying this? :)

Thanks in advance!

**Comment by ericward-noaa** [comment, marked as answer]
Absolutely -- I've done the same -- we just haven't put it into a vignette yet

**Reply by FisheryScience89** [reply]
Thanks
